mmv1/third_party/terraform/tests/resource_dns_record_set_test.go.erb

```diff
@@ -44,10 +44,6 @@ func TestAccDNSRecordSet_basic(t *testing.T) {
 		Steps: []resource.TestStep{
 			{
 				Config: testAccDnsRecordSet_basic(zoneName, "127.0.0.10", 300),
```

Comment:
I believe we should have an import step after every `Config` step to verify values are what they should be.

Reply:
done